### DISCUSSION TITLE
Ability to close videos on web

### DISCUSSION THREAD

**Discussion by Yunhao0799** [photoprism/photoprism#2163] [Q&A] [answered]
When I open a video I cannot close it unless I reload the page

**Comment by lastzero** [comment, marked as answer]
https://docs.photoprism.app/getting-started/troubleshooting/browsers/

**Reply by graciousgrey** [reply]
What browser do you use? 
Does it work for you on our demo: https://demo.photoprism.app/videos

**Reply by Yunhao0799** [reply]
Following the troubleshooting guide, I have successfully identified the extension that creates the conflict (Video Speed Controller in my case). Thanks!
